Swapnil Patle

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "Swapnil Patle",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Lawyer turned **Digital Nomad**. Travel enthusiast Yoga & Peloton aficionado",
```

Comment:
You also need to change the bio here, seems like you have used the one described in the `Quickstart` guide